packages/wallet/backend/src/app.ts

```diff
@@ -169,6 +169,18 @@ export class App {
       transactionController.list
     )
 
+    router.post(
+      '/accounts/:accountId/payment-pointers/:paymentPointerId/register-key',
+      isAuth,
+      paymentPointerController.registerKey
+    )
+
+    router.post(
+      '/accounts/:accountId/payment-pointers/:paymentPointerId/generate-key',
+      isAuth,
+      paymentPointerController.generateKey
+    )
```

Comment:
Why are both `/register-key` and `/generate-key` needed? Would you be able to do everything in a single `/generate-key` call?